base/linalg/dense.jl

```diff
@@ -693,6 +698,198 @@ function inv(A::StridedMatrix{T}) where T
 end
 
 """
+    cos(A::Matrix)
```

Comment:
Perhaps change to `cos(A::AbstractMatrix)` alternatively `cos(A::StridedMatrix)`?